wandb/cli/cli.py

```diff
@@ -1069,6 +1069,22 @@ def launch_sweep(
     if scheduler_args.get("name"):
         name = scheduler_args["name"]
 
+    # configure scheduler job resource
+    resource = scheduler_args.get("resource")
+    if resource:
+        if resource == "local-process" and scheduler_job:
+            wandb.termerror(
+                "Scheduler jobs cannot be run with the 'local-process' resource"
+            )
+            return
+        if resource == "local-process" and scheduler_args.get("docker_image"):
+            wandb.termerror(
+                "Scheduler jobs cannot be run with the 'local-process' resource and a docker image"
+            )
+            return
```

Comment:
Could this be a warning instead of an error? I thought `docker_image` would just be ignored if the resource is `local-process`.

Reply:
interesting, good question, I could see this going either way. If a use explicitly sets a `docker_image`, i'd expect them to want to use it. But if we warn and continue thats probably also a decent outcome. Maybe ben will have more thoughts.

Reply:
I think this should be an error because the sweep config is truly malformed under these conditions. I think we should error and ask them to get it right before they spend $ on this sweep.